framework/src/ppl/delite/framework/ops/DeliteOpsCGen.scala

```diff
@@ -119,6 +119,7 @@ trait CGenDeliteOps extends CGenLoopsFat with GenericGenDeliteOps with CGenDelit
 
   def emitAbstractFatLoopHeader(syms: List[Sym[Any]], rhs: AbstractFatLoop) {
     val kernelName = getKernelName(syms)
+    stream.println("#include \"pcmHelper.h\"")
```

Comment:
Could you point me to the code that generates cpphelperFuncs.h? I'll add '#include pcmHelper.h' to it since cpphelperFuncs.h is included in every kernel's code